extensions/blocks/podcast-player/edit.js

```diff
@@ -61,6 +58,23 @@ const PodcastPlayerEdit = ( {
 	// State.
 	const [ editedUrl, setEditedUrl ] = useState( url || '' );
 	const [ isEditing, setIsEditing ] = useState( false );
+	const [ tracks, setTracks ] = useState( [] );
+
+	// Load RSS feed.
+	useEffect( () => {
+		setTracks( [] );
+		apiFetch( {
```

Comment:
We probably only want to trigger a request when we have a URL, and not before, otherwise we'll get 400 errors when you first add the block to the editor.